mfhdf/xdr/xdrstdio.c

```diff
@@ -129,15 +129,13 @@ xdrstdio_putlong(XDR *xdrs, const long *lp)
 {
     int32_t mycopy;
 
-#if defined(_LP64)
-    if ((*lp > UINT32_MAX) || (*lp < INT32_MIN))
-        return (FALSE);
-#endif
+    if ((*lp > INT32_MAX) || (*lp < INT32_MIN))
```

Comment:
No need to ifdef this - it's always a constraint of XDR and we can use C99 types to check the size